pkg/controller/plan/migration.go

```diff
@@ -1111,6 +1111,20 @@ func (r *Migration) execute(vm *plan.VMStatus) (err error) {
 		if err != nil {
 			return
 		}
+		// only for OVA, fetch config from the conversion pod
+		if r.Source.Provider.Type() == v1beta1.Ova {
```

Comment:
we cant move it to a different stage, because we relying on the pod completion to finish with that stage (we don't have other indication) .

Reply:
I meant to replace this with:
```suggestion
		if vm.Phase == CopyDisksVirtV2V && r.Source.Provider.Type() == v1beta1.Ova {
```
because I think it may fail when we call it during ConvertGuest but if you never saw it failing..

and what about making sure we don't trigger this unnecessarily by checking if the firmware is already set?